README.md

```diff
@@ -71,7 +71,8 @@ Responds with JSON-encoded object with with property **_*article*_**, whose valu
     "author": "butter_bridge",
     "body": "I find this existence challenging",
     "created_at": "2020-07-09T20:11:00.000Z",
-    "votes": 100
```

Comment:
Happy for you to continue doing this if you wish, but this is sort of ticket 13. 
You'd be better updating the file for that ticket, then you would also have completed that ticket when you get to it. 

It would also mean you readme could just say `Available endpoints can be found at WHATEVERTHEURLIS/api`